include/multigrid_pclomp/multi_voxel_grid_covariance_omp.h

```diff
@@ -329,10 +329,6 @@ class MultiVoxelGridCovariance : public pcl::VoxelGrid<PointT>
   {
     sync();
 
-    if (thread_num <= 0) {
-      thread_num_ = 1;
-    }
```

Comment:
Why remove this code?

Reply:
The following line has `thread_num_ = thread_num;` which overwrite the `thread_num_`. Does this code has some side effect to some instances?

Reply:
change `thread_num_` to `thread_num`

Reply:
I fixed it at e71bb76